.gitignore

```diff
@@ -0,0 +1,40 @@
+# intellij ide config folders
+.idea/
```

Comment:
Please only ignore the files that we have in the project and are not necessary. We don't have *idea* folder

Reply:
@kcharles52 .idea/ file is an auto-generated file for the web storm editor. I happen to be using it as my code editor.

Reply:
@kalsmic when contributing to a group project you only add what is relevant to the project. Any custom settings for your personal use are not supposed to be reflected in the project files because they are only specific to your work environment